docs/components/messageListener.md

```diff
@@ -0,0 +1,168 @@
+invoker is triggered only if the component provides it. Each of these invokers accepts the following arguments:
+
+- `tabId`: ID of the currently active tab.
+- `invert`: Whether direction of the given action should be in the opposite order (for thumbsticks).
+- `active`: Whether a new tab or window should be focused when opened.
+- `homepageURL`: URL that a new tab or window should load when opened.
+- `left`: Position of the current browser window from the screen's left edge (in pixels).
+
+The invokers use only those arguments required to perform their specific action.
+
+## Navigation Invokers
+
+### `{messageListener}.openNewTab(active, homepageURL)`
+
+- `active {Boolean}` Whether a new browser tab should be active when opened.
+- `homepageURL {String}` URL that a new browser tab should load when opened.
+- Returns: Nothing.
+
+Opens a new tab in the current browser window using the
+[`chrome.tabs.create`](https://developer.chrome.com/extensions/tabs#method-create) method.
+
+### `{messageListener}.closeCurrentTab(tabId)`
+
+- `tabId {Number}` ID of the currently active tab.
+- Returns: Nothing.
+
+Closes the current tab in the active browser window using the
+[`chrome.tabs.remove`](https://developer.chrome.com/extensions/tabs#method-remove) method.
+
+### `{messageListener}.goToPreviousTab()`
+
+- Returns: Nothing.
+
+Switches from the current tab to the previous tab in the active browser window. If the active tab is the first tab in
+the list, it will switch to the last tab.
+
+### `{messageListener}.goToNextTab()`
+
+- Returns: Nothing.
+
+Switches from the current tab to the next tab in the active browser window. If the active tab is the last tab in the
+list, it will switch to the first tab.
+
+### `{messageListener}.openNewWindow(active, homepageURL)`
+
+- `active {Boolean}` Whether a new browser window should be active when opened.
+- `homepageURL {String}` URL that a new browser window should load when opened.
+- Returns: Nothing.
+
+Opens a new browser window using the
+[`chrome.windows.create`](https://developer.chrome.com/extensions/windows#method-create) method.
+
+### `{messageListener}.closeCurrentWindow()`
+
+- Returns: Nothing.
+
+Closes the currently active browser window using the
+[`chrome.windows.getCurrent`](https://developer.chrome.com/extensions/windows#method-getCurrent) and
+[`chrome.windows.remove`](https://developer.chrome.com/extensions/windows#method-remove) methods.
+
+### `{messageListener}.goToPreviousWindow()`
+
+- Returns: Nothing.
+
+Switches from the currently active browser window to the previous browser window. If the active window is the first
+window in the list, it will switch to the last window.
+
+### `{messageListener}.goToNextWindow()`
+
+- Returns: Nothing.
+
+Switches from the currently active browser window to the next browser window. If the active window is the last window in
+the list, it will switch to the first window.
```

Comment:
I think it would be clearer to vary the wording a bit and not always use "switch".  It's kind of the same as my comments on "action", if you use synonyms or otherwise avoid using the same word over and over, you give people more chance to pick up on a word whose meaning would not otherwise be clear.

In this case you also have the problem of "it" not referring to anything described in the paragraph so far.   I would suggest "focus will move" instead of "it will switch" for all variations on this.